torch/_inductor/compile_fx.py

```diff
@@ -2450,11 +2450,6 @@ def compile_fx(
                 ignore_shape_env=ignore_shape_env,
             )
 
-    if config.deterministic:
-        torch.backends.cudnn.deterministic = True
-        torch.backends.cudnn.benchmark = False
-        torch.backends.mkldnn.deterministic = True  # type: ignore[assignment]
```

Comment:
Remove these since other tests calling torch.use_deterministic_algorithms(True) will fail due to these flags being frozen.

Expect them to be set outside of torch.compile.